miner/worker_test.go

```diff
@@ -134,7 +134,10 @@ func newTestWorkerBackend(t *testing.T, chainConfig *params.ChainConfig, engine
 	default:
 		t.Fatalf("unexpected consensus engine type: %T", engine)
 	}
-	chain, _ := core.NewBlockChain(db, &core.CacheConfig{TrieDirtyDisabled: true}, gspec, nil, engine, vm.Config{}, nil, nil)
+	chain, err := core.NewBlockChain(db, &core.CacheConfig{TrieDirtyDisabled: true}, gspec, nil, engine, vm.Config{}, nil, nil)
+	if err != nil {
+		t.Fatalf("core.NewBlockChain failed: %v", err)
```

Comment:
```suggestion
		t.Fatal(err)
```
I mean, the stack track will tell us that `core.NewBlockChain failed`, right? Not a biggie though

Reply:
No stack trace in the error I was getting:

> [:prompt:] go test -v -run TestGenerateBlock
> === RUN   TestGenerateBlockAndImportEthash
>     worker_test.go:139: core.NewBlockChain failed: genesis not found in chain
> --- FAIL: TestGenerateBlockAndImportEthash (0.00s)
> === RUN   TestGenerateBlockAndImportClique
>     worker_test.go:139: core.NewBlockChain failed: genesis not found in chain
> --- FAIL: TestGenerateBlockAndImportClique (0.00s)
> FAIL
> exit status 1
> FAIL	github.com/ethereum/go-ethereum/miner	0.042s
>

Reply:
Right,  my bad -- no stack trace, but the `worker_test.go:139` shows exactly where the error occurred. But it doesn't really matter, you can leave it as is if you prefer.